src/align.rs

```diff
@@ -87,3 +90,79 @@ def_align!(8, Align8);
 def_align!(16, Align16);
 def_align!(32, Align32);
 def_align!(64, Align64);
+
+/// A `Vec` that uses a 64 byte aligned allocation.
+///
+/// Only works with `Copy` types so that we don't have to handle drop logic.
+#[derive(Default)]
+pub struct AlignedVec64<T: Copy> {
+    inner: Vec<MaybeUninit<Align64<[u8; 64]>>>,
```

Comment:
I thought of a way to make this generic over the alignment by implementing a sealed trait (generic `C` for chunk) for `AlignN<[u8; N]>` and then using `mem::size_of::<C>()` instead of `64`.  I can open a PR for that if you think that's a good idea; I'm not sure how to explain it super well.

Reply:
We decided I'll add this in a follow-up PR instead.